protoc-gen-swagger/genswagger/template.go

```diff
@@ -13,6 +13,49 @@ import (
 	"github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway/descriptor"
 )
 
+// messageToQueryParameters converts a message to a list of swagger query parameters.
+func messageToQueryParameters(message *descriptor.Message, reg *descriptor.Registry, pathParams []descriptor.Parameter) ([]swaggerParameterObject, error) {
+	var parameters []swaggerParameterObject
```

Comment:
It was even better if you make variable names a bit shorter.
https://github.com/golang/go/wiki/CodeReviewComments#variable-names